extension/src/plots/model/index.ts

```diff
@@ -56,7 +56,7 @@ export type CustomPlotsOrderValue = { metric: string; param: string }
 export class PlotsModel extends ModelWithPersistence {
   private readonly experiments: Experiments
 
-  private nbItemsPerRow: Record<PlotsSection, number>
+  private nbItemsPerRowOrWidth: Record<PlotsSection, number>
```

Comment:
[Q] Do these need to be stored together?

Reply:
Was debating this, but they function the exact same way it's just that one cannot be called `nbItemsPerRow`.  Separating them would probably mean checking whether it has one property or another multiple times.